@here/harp-mapview/lib/Utils.ts

```diff
@@ -944,78 +944,77 @@ export namespace MapViewUtils {
         worldTarget: THREE.Vector3,
         camera: THREE.PerspectiveCamera
     ): number {
-        const cameraRotationMatrix = new THREE.Matrix4();
-        cameraRotationMatrix.extractRotation(camera.matrixWorld);
-        const screenUpVector = new THREE.Vector3(0, 1, 0).applyMatrix4(cameraRotationMatrix);
-        const screenSideVector = new THREE.Vector3(1, 0, 0).applyMatrix4(cameraRotationMatrix);
-        const screenVertMidPlane = new THREE.Plane().setFromCoplanarPoints(
-            camera.position,
-            worldTarget,
-            worldTarget.clone().add(screenUpVector)
-        );
-        const screenHorzMidPlane = new THREE.Plane().setFromCoplanarPoints(
-            camera.position,
-            worldTarget,
-            worldTarget.clone().add(screenSideVector)
-        );
-
-        const cameraPos = cache.vector3[0];
-        cameraPos.copy(camera.position);
-
-        const halfVertFov = THREE.MathUtils.degToRad(camera.fov / 2);
-        // TODO: Support off-center projections.
-        const halfHorzFov = calculateHorizontalFovByVerticalFov(halfVertFov * 2, camera.aspect) / 2;
-
-        // tan(fov/2)
-        const halfVertFovTan = 1 / Math.tan(halfVertFov);
-        const halfHorzFovTan = 1 / Math.tan(halfHorzFov);
-
-        const cameraToTarget = cache.vector3[1];
-        cameraToTarget.copy(cameraPos).sub(worldTarget).negate();
-
-        const cameraToTargetNormalized = new THREE.Vector3().copy(cameraToTarget).normalize();
-
-        const offsetVector = new THREE.Vector3();
-
-        const cameraToPointOnRefPlane = new THREE.Vector3();
-        const pointOnRefPlane = new THREE.Vector3();
-
-        function checkAngle(
-            point: THREE.Vector3,
-            referencePlane: THREE.Plane,
-            maxAngle: number,
-            fovFactor: number
-        ) {
-            referencePlane.projectPoint(point, pointOnRefPlane);
-            cameraToPointOnRefPlane.copy(cameraPos).sub(pointOnRefPlane).negate();
-
-            const viewAngle = cameraToTarget.angleTo(cameraToPointOnRefPlane);
-
-            if (viewAngle <= maxAngle) {
-                return;
-            }
-
-            const cameraToPointLen = cameraToPointOnRefPlane.length();
-            const cameraToTargetLen = cameraToTarget.length();
-
-            const newCameraDistance =
-                cameraToPointLen * (Math.sin(viewAngle) * fovFactor - Math.cos(viewAngle)) +
-                cameraToTargetLen;
-
-            offsetVector
-                .copy(cameraToTargetNormalized)
-                .multiplyScalar(cameraToTargetLen - newCameraDistance);
-
-            cameraPos.add(offsetVector);
-            cameraToTarget.sub(offsetVector);
-        }
-
+        //                camY
+        //       targetDist^
+        //      |<-------->|     Ps
+        //     constD pEyeZ|    /|  ^
+        //      |<-->|<--->|   / |  |
+        //      |    |     |  /  |  | |ndcY-O.y|*h/2
```

Comment:
please document what h is to remove any potential ambiguity

Reply:
done